src/uncrustify.cpp

```diff
@@ -104,7 +104,7 @@ static bool ends_with(const char *filename, const char *tag, bool case_sensitive
  * @param no_backup    don't create a backup, if filename_out == filename_in
  * @param keep_mtime   don't change the mtime (dangerous)
  */
-static void do_source_file(const char *filename_in, const char *filename_out, const char *parsed_file, bool no_backup, bool keep_mtime);
+static void do_source_file(const char *filename_in, const char *filename_out, const char *parsed_file, const char *token_output_file, const char *ptoken_output_file, bool no_backup, bool keep_mtime);
```

Comment:
I'd just use the previous `parsed_file` but use a new export format or something like this (more on this later).